peachjam/views/robots.py

```diff
@@ -0,0 +1,19 @@
+from django.views.generic import TemplateView
+
+from peachjam.models import CoreDocument
+
+
+class RobotsView(TemplateView):
+    template_name = "peachjam/robots.txt"
+    content_type = "text/plain"
+
+    def get(self, request, *args, **kwargs):
```

Comment:
You're only adding context, so overriding get_context_data() is the preferred way of doing this -- no need to override `get()`.